### ISSUE TITLE
Use of platform.dist() in configure breaks python 3.8 compatibility

### ISSUE THREAD

**Issue by risicle** [johanmattssonm/xmlbird#9]
`platform.dist()` was deprecated in python 3.5 and removed in 3.8 (https://docs.python.org/3.7/library/platform.html#platform.dist). Unfortunately I don't have an alternative to give you.

Patching it out allows successful installation with python 3.8.

**Comment by johanmattssonm** [comment]
Thank you for the report. Will this workaround work?

```
def is_debian():
  try:
  	version = platform.version()
  	print("OS Version: " + version)
  	
  	if version.find("Ubuntu") > -1:
  		return True

  	if version.find("Debian") > -1:
  		return True
  		
  except:
    return False
    
  return False
```

**Comment by risicle** [comment]
Probably - though of course I can't guarantee anything.

I'm currently in the process of fixing the nixos package for this so all I'm doing is patching it out with a string constant.

**Comment by johanmattssonm** [comment]
Thank you. I will stick to the platform.version workaround unless someone finds an error. A new version has been released (1.2.12).
